latex/content/debug.tex

```diff
@@ -1,6 +1,6 @@
 \subsection{Debug}
 
-\begin{frame}[fragile]{\lstinline+\\overfullrule+}
+\begin{frame}[fragile]{\texttt{\textbackslash overfullrule}}
```

Comment:
Or we change, that inline code is dispayed white instead of red in the title only.